spacemacs.org

```diff
@@ -0,0 +1,97 @@
+best of Vim and Emacs, and it does so with a focus on ease of use and
+productivity.
+
+I had been thinking about trying either Vim or Emacs for a while. I liked
+the idea of Vim's keybinding and learning them so I could have a text editor
+available on whatever machine I use.
+
+But I had also heard a ton about how many great things you can do completely
+inside Emacs. Given I started out as a Java developer using [[https://eclipse.org/][Eclipse]], I like
+the idea of being able to do a lot inside one same tool.
+
+*** Ergonomic
+
+Spacemacs consistently uses Vim key bindings for all interactions with the
+editor. I've asked a few Vim users and they all agree it has the best Vim
+emulation around.
+
+Commands are also mnemonic and therefore easy to remember:
+
+- To save - ~SPACE f s~ (space file save)
+- To switch buffer - ~SPACE b n~ (space buffer next)
+- To split windows - ~SPACE w 2~ (space window two)
+
+*** No set-up!
+
+Spacemacs comes with all the best Emacs packages already installed and
+pre-optimized.
+
+And it looks really good, no need to fiddle with the themes to make it pretty.
+
+*** Easy to learn
+
+New tools have learning curves. Emacs is particularly infamous for having a
+steep learning curve.
+
+Most new Spacemac users are surprised to find it has a pleasantly low learning
+curve.
+
+Whenever you start a Spacemacs command (by pressing ~SPACE~) a pop-up menu
+appears listing all possible keys you could press from there. Help arrives
+just in time, not a moment before or after.
+
+** Why I left Sublime
+
+I started my days as a javascript developer using [[https://www.sublimetext.com/][Sublime Text]]. I had a lot of
+fun with Sublime, and a still keep it around on my machine because I love how it
+does [[https://youtu.be/WXuBgSpLpK4?t=58s][multiple cursors]].
+
+*** Too many windows
+
+However, at Curiosity Media we structure our apps around a service-oriented
+architecture. In fact, a website like [[http://www.spanishdict.com/traductor/][SpanishDict]] has around 15 separate
+services we leverage.
+
+How I used Sublime was fairly typical, I would type ~subl .~ from my terminal
+to launch a new Sublime window at the current working directory. If I needed
+to do some work in another repo, I launched a new window. Soon I ended up
+with a lot of windows, and with no good way to flip between them quickly.
+
+*** Navigate the file tree
+
+I navigate files in two main ways with Spacemacs. ~SPACE p p~ let's you fuzzy
+match any directory on your system, then any file within that directory.
+And `SPACE f t` opens up a file system explorer that's extremely quick and easy
+to navigate.
+
+*** Buffer editing is better
+
+Vim and Emacs use buffers. A buffer is a file loaded into memory so that you
+can edit it. Buffers are infinitely better than Sublime tabs. You can switch
+between them quicker and you can open as many as you want (have you ever had
+10+ Sublime tabs open? They get hard to read).
```

Comment:
https://github.com/travmik/ZenTabs is probably even better for both aspects of "too many tabs" problem in ST